### PR TITLE
Tweak the copy a little bit

### PR DESCRIPTION
**Pull request by mitechie**
This tweaks the copy a little bit. It attempts to de-emphasize JAAS the controller, focus the wording a bit, and phrase things slightly more technically correct.

### CODE REVIEW

File: src/en/getting-started-jaas.md [hunk 1]
```diff
@@ -244,9 +244,8 @@ you can manually copy and paste the unique authorisation URL from the command
 output.
 
 After successful authentication, you will be asked to enter a descriptive name
-for the JAAS controller, giving you access to the same controller used by the
-JAAS web interface. This means any models or applications you have already
-deployed are now accessible from the command line.
+for the JAAS controller. This means any models or applications you have
+already deployed are now accessible from the command line.
```
Comment (pmatulis) [inline src/en/getting-started-jaas.md]: Providing a name for the controller does not mean models/applications are now accessible from the CLI. Maybe just:

"Now|Henceforth, any models or applications..."

**Comment by mitechie** [comment]
Thank you for the feedback. Updated the three items. PTAL

File: src/en/getting-started-jaas.md [hunk 2]
```diff
@@ -244,8 +244,8 @@ you can manually copy and paste the unique authorisation URL from the command
 output.
 
 After successful authentication, you will be asked to enter a descriptive name
-for the JAAS controller. This means any models or applications you have
-already deployed are now accessible from the command line.
+for the JAAS controller. This means you can operate your models or
+applications in JAAS from the command line.
```
Comment (pmatulis) [inline src/en/getting-started-jaas.md]: Sorry to continue with this but what does "This means" refer to? I still think you should break the logic here and just say:

"Now|At-this-time|Henceforth|Going-forward|Once-authenticated, you can..."

Comment (mitechie) [inline src/en/getting-started-jaas.md]: No apology needed. I didn't look at it right from your last comment. I've updated it and agree.

**Pull request merged**